### ISSUE TITLE
[Accessibility] DetailsList's checkboxes have color contrast accessibility issue.

### ISSUE THREAD

**Issue by saurabhsuniljain** [OfficeDev/office-ui-fabric-react#4269]
#### Describe the issue:

The checkbox in the DetailsList has Color-Contrast Issue:
I have used the DetailsList and did accessibility testing on it using KerosForWeb extension and found the following issue:

> Issue Details: Elements must have sufficient [color contrast](https://dequeuniversity.com/rules/axe/2.6/color-contrast)

> Element has insufficient color contrast of 1.67 (foreground color: #c8c8c8, background color: #ffffff, font size: 13.5pt, font weight: normal). Expected contrast ratio of 4.5:1

#### Actual behavior:

The checkbox has color-contrast accessibility issue.

#### Expected behavior:

The checkbox should not have color contrast issue.

**Comment by betrue-final-final** [comment]
What state was the check box in? Is this hover?

**Comment by saurabhsuniljain** [comment]
- Normal State: 
>Element has insufficient color contrast of 1.67 (foreground color: #c8c8c8, background color: #ffffff, font size: 13.5pt, font weight: normal). Expected contrast ratio of 4.5:1
- Focus State:
>Element has insufficient color contrast of 1.09 (foreground color: #ffffff, background color: #eff6fc, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1
- Hover State
>Element has insufficient color contrast of 1.52 (foreground color: #c8c8c8, background color: #f4f4f4, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

**Comment by aditima** [comment]
@betrue-final-final could you please assign to @jordandrako once we have clarity on the fix?

**Comment by betrue-final-final** [comment]
Sure. @jordandrako, here's the info.
Rest has no visible check. 
Hover, and key focus, the border is #666666, or neutral secondary. The check is only supporting info in this case, but is tripping Keros because it's lighter. However the empty circle is clear enough.

Please check the border, and if it is #666666, then close this issue.

**Comment by betrue-final-final** [comment]
One thing we could do is stop showing the circle on hover areas which do not result in a selection, and only show it without the check on hover over areas that will result in a selection. We can chat tomorrow.
